app/templates/components/forms/wizard/sponsors-step.hbs

```diff
@@ -85,7 +85,7 @@
       {{t 'Save draft'}}
       <i class="save icon"></i>
     </button>
-    {{#if data.event.locationName}}
+    {{#if (and data.event.locationName data.event.tickets.length)}}
```

Comment:
@shreyanshdwivedi Just a doubt , If user has not added any ticket , And he tries to publish the event , how is he going know that where is he going wrong?

Reply:
I also go the same thought. But we are doing the same in case for location too.
As I mentioned earlier - 
```
For tickets, we cannot simply raise error message from server if user has no tickets associated because save function on tickets is called after event is saved to db (as it requires event-id).
```
However if you anything in mind which we can do on the front-end itself, please share

Reply:
@shreyanshdwivedi Can't we  have an `Ember Validator` for location and tickets ?

Reply:
Since they are not simple input fields, I'll have to check if we can implement Ember Validation here or not.

Reply:
I don't find a way to implement it